app/views/ReportingFrequencyView.scala.html

```diff
@@ -307,21 +309,76 @@ <h2 class="govuk-summary-card__title" id="manage-reporting-obligations-card-head
     }
 }
 
+@yourReportingObligations = {
+    @h2(msg = "reporting.frequency.your.obligations.heading", optId = Some("your-obligations-heading"))
+    @p(id = Some("your-obligations-p1")){
+        @messages("reporting.frequency.your.obligations.p1")
+    }
+
+    @p(id = Some("your-obligations-updates-and-deadlines")){
+        @messages("reporting.frequency.your.obligations.p2")
+        @linkComponent(nextUpdatesLink,
+            message = messages("reporting.frequency.your.obligations.link.text"),
+            rel=Some("noreferrer noopener"),
+            id = Some("your-obligations-updates-and-deadlines-link")
+        )
+        @messages("reporting.frequency.your.obligations.link.text.additional.english")
+    }
+}
+
+@mandatoryReportingOnlyExempt = {
+    @h2(msg = "reporting.frequency.mandatoryReporting.heading.R17", optId = Some("mandatory-reporting-heading"))
+
+    @p(id = Some("mandatory-reporting-text")){
+        @messages("reporting.frequency.mandatoryReporting.onlyExempt.p1")
+    }
+
+    @p(id = Some("mandatory-reporting-text-2")) {
+        @messages("reporting.frequency.mandatoryReporting.onlyExempt.p2")
+        @linkComponent(ExternalUrlHelper.saWhoNeedsToSignUpUrl, message = s"${messages("reporting.frequency.mandatoryReporting.onlyExempt.link.text")} ", rel = Some("noreferrer noopener"), target = Some("_blank"), id = Some("mandatory-reporting-link"), additionalMessage = Some(messages("base.fullstop")))
+    }
+}
+
+@mandatoryReportingHasExempt = {
+    @h2(msg = "reporting.frequency.mandatoryReporting.heading.R17", optId = Some("mandatory-reporting-heading"))
+
+    @p(id = Some("mandatory-reporting-text")){
+        @messages("reporting.frequency.mandatoryReporting.hasExempt.p1")
+    }
+
+    @govukInsetText(InsetText(id = Some("mandatory-reporting-inset"), content = HtmlContent(messages("reporting.frequency.mandatoryReporting.inset.R17"))))
+
+    @p(id = Some("mandatory-reporting-text-2")){
+        @messages("reporting.frequency.mandatoryReporting.hasExempt.p2")
+    }
+
+    @p(id = Some("mandatory-reporting-text-3")) {
+        @messages("reporting.frequency.mandatoryReporting.hasExempt.p3")
+        @linkComponent(ExternalUrlHelper.saWhoNeedsToSignUpUrl, message = s"${messages("reporting.frequency.mandatoryReporting.hasExempt.link.text")} ", rel = Some("noreferrer noopener"), target = Some("_blank"), id = Some("mandatory-reporting-link"), additionalMessage = Some(messages("base.fullstop")))
+    }
+}
+
```

Comment:
There's a lot of duplicate sections here between the two mandatory reporting sections.

If you can could we combine the two and just have a conditional based on whether its only exempt vs some exemptions